db/migrate/20240610061157_fix_parent_id_format.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class FixParentIdFormat < ActiveRecord::Migration[7.1]
+  def change
+    # rubocop:disable Rails/ReversibleMigration
+    change_column :task_files, :parent_id, :bigint, limit: nil
+    change_column :model_solutions, :parent_id, :bigint, limit: nil
+    change_column :tests, :parent_id, :bigint, limit: nil
+    # rubocop:enable Rails/ReversibleMigration
```

Comment:
You could consider marking this migration as [`up_only`](https://apidock.com/rails/ActiveRecord/Migration/up_only) or splitting it in dedicated methods [`up` and `down`](https://edgeguides.rubyonrails.org/active_record_migrations.html#making-the-irreversible-possible).